libmscore/scorediff.cpp

```diff
@@ -211,15 +211,15 @@ int MscxModeDiff::adjustSemanticsMscxOneDiff(std::vector<TextDiff>& diffs, int i
       int iScore;
       switch(diff->type) {
             case DiffType::EQUAL:
-                  /* FALLTROUGH */
+                  /* FALLTHROUGH */
```

Comment:
Maybe use  `Q_FALLTHROUGH();` here instead?

Reply:
Will change. BTW, there are also instances of `FALLTHROUGH` in `libmscore/page.cpp` & `libmscore/pitchspelling.cpp`  
Shall I change those as well ?

Reply:
Not sure (I just brought that up in the developers' Telegram chat), at least those are not typos, and those in page.cpp won't work with a trailing `;`

Reply:
Actually in page.cpp it doesn't work with `;` (as that would change the semanitic) and on MSVC not without either (compiler error C2143). So better leave it as is. Changing pitchspelling.cpp should be OK though.

Reply:
@Jojo-Schmitz done in c975b7d